packages/xo-web/src/xo-app/settings/audit/index.js

```diff
@@ -371,6 +378,19 @@ export default decorate([
             {_('auditCheckIntegrity')}
           </ActionButton>
         </div>
+        {state.isUserActionsRecordInactive && (
+          <Link
+            className='text-warning'
+            to={{
+              pathname: '/settings/plugins',
+              query: {
+                s: 'id%3A%2F^audit%24%2F',
+              },
+            }}
+          >
+            <Icon icon='alarm' /> {_('auditInactiveUserActionsRecord')}
+          </Link>
+        )}
```

Comment:
Wrap it with a `p`.

Reply:
I did it before using the `span` i didn't like the space above. But i you see if the space is need i'll wrap it.

Reply:
It makes more sense semantically and in our case, it also prevents the loading spinner of the table from being on the same line (it's probably an inline element for some reason). The space isn't a problem IMO.